net/iperf3/Portfile

```diff
@@ -42,13 +42,12 @@ if {${subport} eq ${name}} {
 }
 
 subport ${name}-devel {
-    github.setup        esnet iperf 2a7847a08154d5d224d1836e86afa06d26b512c5
-    version             20190621
-    revision            2
+    github.setup        esnet iperf 02a5f4755878b319f0db5ccd490daf61e6d76043
+    version             20191101
 
```

Comment:
For the future, when resetting the revision, we decided to leave a ```revision 0``` behind for future use for an automated tool to revbump things.

Reply:
Thanks @kencu. I'll make a note of it.